web/src/themes/awbrn.css

```
@@ -89,72 +89,72 @@
-      --duration-slow-max: 935ms;
+      --text-display-3-leading: 1.2549;
+      --duration-fast-min: 65ms;
+      --duration-fast: 90ms;
+      --duration-fast-max: 130ms;
+      --duration-medium-min: 105ms;
+      --duration-medium: 150ms;
+      --duration-medium-max: 215ms;
+      --duration-slow-min: 365ms;
+      --duration-slow: 520ms;
+      --duration-slow-max: 745ms;
       --font-family-body: Nunito, system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected "Nunito" to be "nunito" (value-keyword-case)

(value-keyword-case)